wagtail_transfer/locators.py

```diff
@@ -95,6 +107,9 @@ def __init__(self, model, fields):
         self.model = model
         self.fields = fields
 
+    def get_or_create_uid_for_local_id(self, id):
+        return self.get_uid_for_local_id(id)
```

Comment:
Is there anywhere this is currently being used? I think we should either make this create the object, or leave it unimplemented

Reply:
This can't create the object - there is no IDMapping object to create for a field lookup. When importing, a locator is retrieved and then the `get_or_create_uid_for_local_id(self, id)` is called on it (used to be `get_uid_for_local_id`, but I didn't want the Page Chooser creating random ids all over the place just when looking at a page)

Reply:
I've reverted this separation for now as we talked about